.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Hurts to see first PR to just ignore someone's codestyle and force your own 🙄 
It works, why change it.

Reply:
Single quote and comma tangle is what we use in our current repos, it's not my codestyle it's Busy codestyle.

Reply:
I will revert it